scripts/update-xliff.py

```diff
@@ -50,15 +50,23 @@ def main():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('base_folder', help='Path to folder including subfolders for all locales')
+    parser.add_argument('locales', default='*', nargs='*', help='Locales to process (default: %(default)s)')
```

Comment:
This change looks good, I am just a little worried that a argument value of `*` is a bit odd in UNIX terms. It needs to be escaped in the shell otherwise it expands to all files in your current directory.

Or is this not an issue and is `*` just use to recognize the default if the argument is not provided?